src/lib/common/FileInput.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+  export let label: string;
+  export let onChange: (e: Event) => void;
+
+  // TODO Only one component per page is supported, because the ID is
+  // hardcoded. This is only necessary for the Playwright test.
+  let fileInput: HTMLInputElement;
+</script>
+
+<!-- TODO Interactive elements inside a label are apparently invalid, but this works -->
```

Comment:
This is an old hack, probably from Stackoverflow, to style the file input box reasonably